bpaotu/bpaotu/frontend/src/pages/search_page/index.tsx

```diff
@@ -30,18 +31,21 @@ export const SearchPage = props => (
     </Row>
 
     <Row className="space-above">
-      <Col sm={{ size: 2, offset: 5 }}>
-        <SearchButton isDisabled={props.isSearchButtonDisabled} search={props.search} />
-      </Col>
+      {props.isSearchButtonDisabled
```

Comment:
It probably makes sense to remove `isSearchButtonDisabled` to `isSearchInProgress`